src/app/components/iterations-panel/iterations-panel.component.ts

```diff
@@ -29,27 +33,48 @@ export class IterationComponent implements OnInit, OnDestroy, OnChanges {
 
   @Input() takeFromInput: boolean = false;
   @Input() iterations: IterationModel[] = [];
+  @Input() collection = [];
+
+  @ViewChild('modal') modal: FabPlannerIterationModalComponent;
+  @ViewChild('treeList') treeList: TreeListComponent;
+  @ViewChild('treeListItemTemplate') treeListItemTemplate: TemplateRef<any>;
+  @ViewChild('treeListLoadTemplate') treeListLoadTemplate: TemplateRef<any>;
+  @ViewChild('treeListTemplate') treeListTemplate: TemplateRef<any>;
+  @ViewChild('treeListItem') treeListItem: TreeListComponent;
+
 
   authUser: any = null;
   loggedIn: Boolean = true;
   editEnabled: Boolean = false;
   isBacklogSelected: Boolean = true;
-  isCollapsedIteration: Boolean = false;
-  isCollapsedCurrentIteration: Boolean = false;
-  isCollapsedFutureIteration: Boolean = true;
-  isCollapsedPastIteration: Boolean = true;
+  //isCollapsedIteration: Boolean = false;
+  //isCollapsedCurrentIteration: Boolean = false;
+  //isCollapsedFutureIteration: Boolean = true;
+  //isCollapsedPastIteration: Boolean = true;
   barchatValue: number = 70;
   selectedIteration: IterationModel;
   allIterations: IterationModel[] = [];
-  futureIterations: IterationModel[] = [];
-  currentIterations: IterationModel[] = [];
-  closedIterations: IterationModel[] = [];
+  //futureIterations: IterationModel[] = [];
+  //currentIterations: IterationModel[] = [];
+  //closedIterations: IterationModel[] = [];
   eventListeners: any[] = [];
   currentSelectedIteration: string = '';
   dragulaEventListeners: any[] = [];
+  masterIterations;
+  treeIterations;
 
   private spaceSubscription: Subscription = null;
 
+  // See: https://angular2-tree.readme.io/docs/options
+  treeListOptions = {
+    allowDrag: false,
+    levelPadding: 30,
+    allowDrop: (element, to) => {
```

Comment:
If you don't allow dragging, you probably don't need to specify an allowDrop condition.